lib/models/selected_files.dart

```diff
@@ -5,7 +5,7 @@ import 'package:photos/events/clear_selections_event.dart';
 import 'package:photos/models/file.dart';
 
 class SelectedFiles extends ChangeNotifier {
-  final files = <File>{};
+  final files = <File>[];
```

Comment:
Changing this will cause performance degradation while selecting/updating the states while selecting large number of elements. I don't know if we should make this change just for collage (which will be active for just 6 selections)